starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html

```diff
@@ -1,14 +1,14 @@
 <h1 class="heading">Deferrable Views</h1>
 
 <p>
-  Deferrable views can be used in component template to defer the loading of
+  Deferrable views can be used in a component template to defer the loading of
   select dependencies within that template. Those dependencies include
   components, directives, and pipes, and any associated CSS. To use this
   feature, you can declaratively wrap a section of your template in a
   <code>&#64;defer</code> block which specifies the loading conditions.
   Deferrable views support a series of triggers, prefetching, and several sub
   blocks used for placeholder, loading, and error state management. You can also
-  create custom conditions with when and prefetch when.
+  create custom conditions with <code>when</code> and <code>prefetch when</code>.
```

Comment:
Is this correct -- those are literals?

I probably have this wrong, but it needs some revision as it stands.  Help!  :)

Reply:
cc @9kubczas4

Reply:
Exactly, the proposed change is correct

Reply:
Thanks!